### ISSUE TITLE
Can you release the code to compute the evaluate metrics of the output results?

### ISSUE THREAD

**Issue by cczka** [XiaoyuanYi/StyIns#3]
Hi! Thanks for the great work on the text style transfer. I computed the bleu score of your output results. It's 49.52 and seems far higher than what is reported in your paper. So, I really want to konw how you compute the blue score. Thanks once again.

**Comment by naveen-kinnal** [comment]
Hello @cczka . I am also working on this paper. Could you please let me know if you executed this for YELP? I am getting the issue as

`ValueError: Sample larger than population or is negative`

Did you get something like this?

**Comment by cczka** [comment]
> Hello @cczka . I am also working on this paper. Could you please let me know if you executed this for YELP? I am getting the issue as
> 
> `ValueError: Sample larger than population or is negative`
> 
> Did you get something like this?

yes, it's the problem about the dataloader. You can set the batchsize to a smaller value.

**Comment by XiaoyuanYi** [comment]
Hi, sorry for my late reply due to some personal issues during last month. 
(1) @naveen-kinnal  For the sampler problem, just as @cczka mentioned, you can set a smaller batch size since we need to sample kb sentences for each training iteration, where k is the instance size, and b is batch size. When the size of your training/validation/testing sets is smaller than k*b, this problem will occur.

(2)  @cczka  For the bleu score, 
1. have you processed your data (including testing inputs and references) as described in Sec.2 of readme?
2. On Yelp, we use only a single reference instead of multiple references, as some previous papers do for calculating bleu. Would you please make sure you do the same since multiple references would increase the bleu value?
3. In the early training phase, or the adversarial training is not well balanced, the model may achieve higher bleu scores but worse style accuracy. Would you please check if you get reasonable accuracy?

If you have any other questions, just feel free to comment, open new issues or email me.

**Comment by stone-cwang** [comment]
Hi! Thanks for the great work on the text style transfer. I would like to ask how to modify the code to add multiple styles? Can you give some hints?
